daprdocs/content/en/operations/configuration/invoke-allowlist.md

```diff
@@ -233,7 +234,78 @@ spec:
     trustDomain: "myDomain"
 ```     
 
-For example, this is how the pythonapp is deployed to Kubernetes in the default namespace with this configuration file.
+### Self-hosted mode
+This example uses the [hello world](https://github.com/dapr/quickstarts/tree/master/hello-world/README.md) quickstart.
+
+The following steps run the Sentry service locally with mTLS enabled, set up necessary environment variables to access certificates, and then launch both the node app and python app each referencing the Sentry service to apply the ACLs.
+
+ 1. Follow these steps to run the [Sentry service in self-hosted mode]({{< ref "mtls.md" >}}) with mTLS enabled.
+ 2. In a command prompt, set these environment variables  
+{{< tabs Linux/MacOS Windows>}}
+
+{{% codetab %}}
+```bash
+export DAPR_TRUST_ANCHORS=`cat $HOME/.dapr/certs/ca.crt`
+export DAPR_CERT_CHAIN=`cat $HOME/.dapr/certs/issuer.crt`
+export DAPR_CERT_KEY=`cat $HOME/.dapr/certs/issuer.key`
+export NAMESPACE=default
+ ```
+{{% /codetab %}}
+
+{{% codetab %}}
+```powershell
+set DAPR_TRUST_ANCHORS=``
+set DAPR_CERT_CHAIN=``
+set DAPR_CERT_KEY=``
+set NAMESPACE=default
+```
+{{% /codetab %}} 
+ 
+3. Run daprd to launch a Dapr sidecar for the node.js app with mTLS enabled, referencing the local Sentry service.
+```
+daprd --app-id nodeapp --dapr-grpc-port 50002 -dapr-http-port 3501 --log-level debug --app-port 3000 --enable-mtls --sentry-address localhost:50001 --config nodeappconfig.yaml
+ ```
+4. Run the node app in a separate command prompt.
+ ```
+ node app.js
+```
+5. In another command prompt, set these environment variables.  
+{{< tabs Linux/MacOS Windows>}}
+
+{{% codetab %}}
+```bash
+export DAPR_TRUST_ANCHORS=`cat $HOME/.dapr/certs/ca.crt`
+export DAPR_CERT_CHAIN=`cat $HOME/.dapr/certs/issuer.crt`
+export DAPR_CERT_KEY=`cat $HOME/.dapr/certs/issuer.key`
+export NAMESPACE=default
+ ```
+{{% /codetab %}}
+
+{{% codetab %}}
+```powershell
```

Comment:
Why are the Windows variables empty? Do they just need to be filled in?

Reply:
I added the powershell commands